setup.cfg

```diff
@@ -0,0 +1,58 @@
+[metadata]
+name = stdpopsim
+author = PopSim Consortium
+license = GNU GPLv3+
+description = A library of population genetic simulation models
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/popsim-consortium/stdpopsim
+author_email = popgen_benchmark@lists.uoregon.edu #TODO: Change to a better address?
+classifiers =
+    Development Status :: 4 - Beta
+    Intended Audience :: Developers
+    License :: OSI Approved :: GNU General Public License v3 or later (GPLv3+)
+    Operating System :: POSIX :: Linux
+    Operating System :: Microsoft :: Windows
+    Operating System :: MacOS :: MacOS X
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
+    Topic :: Scientific/Engineering
+    Topic :: Scientific/Engineering :: Bio-Informatics
+keywords = simulations, selection, recombination map, models
+project_urls =
+    Documentation = https://popsim-consortium.github.io/stdpopsim-docs/stable/index.html
+    Source Code = https://github.com/popsim-consortium/stdpopsim
+    Bug Tracker = https://github.com/popsim-consortium/stdpopsim/issues
+
+[options]
+packages = stdpopsim
+zip_safe = False # TODO: needed? we're not using mypy
```

Comment:
Not setting this to False can have surprising consequences (including for downstream packages that depend on stdpopsim), so I reckon just keep this.